python/social_bot/envs/simple_navigation.py

```diff
@@ -91,7 +91,9 @@ def __init__(self,
         task_group = teacher.TaskGroup()
         task = GoalTask(
             env=self,
+            max_steps=120,
```

Comment:
Default max_steps for simple-navigation whose range is smaller has been set to 120.